### PR TITLE
add AIPodNav to list of app types

### PR DESCRIPTION
**Pull request by liuyunwushui**

### CODE REVIEW

File: server/data/apps.json [hunk 1]
```diff
@@ -27,6 +27,53 @@
...
+        "elementURL": "https://www.aipodnav.com/discover/best-podcasts"
+      },
+      {
+        "elementName": "mind map",
+        "elementURL": "https://www.aipodnav.com/"
+      },
+      {
+        "elementName": "summary",
+        "elementURL": "https://www.aipodnav.com/discover/new-ai-podcast-summary"
+      },
```
Comment (Marzal) [inline server/data/apps.json:62]: Hi, this are not Podcasting 2.0 tags

Comment (liuyunwushui) [reply server/data/apps.json]: Alright, I have resubmitted it.

Comment (Marzal) [reply server/data/apps.json]: I have took a look at your project, but I don't find any reference to the `episodes tag` in the link you provided

In what way are you using it?
https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/1.0.md#episode

**Comment by liuyunwushui** [comment]
Hello, how long does it take for it to be displayed on the web page after merging?

**Comment by Marzal** [comment]
> Hello, how long does it take for it to be displayed on the web page after merging?

It's a manual procedure, @daveajones publish all the changes when enough PR are merged on a regular basis (days or 1/2 weeks).

**Comment by daveajones** [comment]
Deployed.

**Pull request merged**